components/deck/EditDeck.js

```diff
@@ -87,7 +92,7 @@ export default ({
         />
         <ButtonsWrapper>
           <Button type="button" onClick={submitChanges} disabled={!edited}>
-            Save changes
+            {edited ? 'Save changes' : 'All changes are saved'}
           </Button>
           <Link href="/repeat">
             <Button type="button" cancel>
```

Comment:
We should have another name for this button, maybe back or similar

Reply:
I was thinking that the next step might be to have a view where you only show the deck, without editing it. Like this:

1. View decks -> click on a deck
2. View a deck and the words in it -> click 'run test' or 'edit'
  a) Run test is the view where you guess words.
  b) Edit is the EditDeck view.

In this scenario, the cancel button would take you back to the view deck view.

What do you think about this? Should we rename it go back for now and change it later?